frontend/src/old-pages/Configure/SlurmMemorySettings.tsx

```diff
@@ -0,0 +1,110 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License"). You may not use this file except in compliance
+// with the License. A copy of the License is located at
+//
+// http://aws.amazon.com/apache2.0/
+//
+// or in the "LICENSE.txt" file accompanying this file. This file is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES
+// OR CONDITIONS OF ANY KIND, express or implied. See the License for the specific language governing permissions and
+// limitations under the License.
+
+import * as React from 'react'
+import {Trans, useTranslation} from 'react-i18next'
+import {
+  Container,
+  Header,
+  Alert,
+  Toggle,
+  SpaceBetween,
+} from '@awsui/components-react'
+import {setState, getState, useState, clearState} from '../../store'
+
+function SlurmMemorySettings() {
+  const {t} = useTranslation()
+  const [infoAlertVisible, setInfoAlertVisible] = React.useState(false)
+  const slurmSettingsPath = [
+    'app',
+    'wizard',
+    'config',
+    'Scheduling',
+    'SlurmSettings',
+  ]
+  const memoryBasedSchedulingEnabledPath = [
+    ...slurmSettingsPath,
+    'EnableMemoryBasedScheduling',
+  ]
+  const memoryBasedSchedulingEnabled = useState(
+    memoryBasedSchedulingEnabledPath,
+  )
+
+  const toggleMemoryBasedSchedulingEnabled = () => {
+    const setMemoryBasedSchedulingEnabled = !memoryBasedSchedulingEnabled
+    if (setMemoryBasedSchedulingEnabled)
+      setState(
+        memoryBasedSchedulingEnabledPath,
+        setMemoryBasedSchedulingEnabled,
+      )
+    else {
+      clearState(memoryBasedSchedulingEnabledPath)
+      if (Object.keys(getState([...slurmSettingsPath])).length === 0)
+        clearState([...slurmSettingsPath])
+    }
+  }
+
+  return (
+    <Container
+      header={
+        <Header variant="h2">
+          <SpaceBetween size="xs" direction="horizontal">
+            {t('wizard.queues.slurmMemorySettings.container.title')}
+            <a
+              href="#"
+              onClick={() => setInfoAlertVisible(true)}
```

Comment:
issue: the Info button on the container title should point to an explanation of the feature itself (see previous Info button in the Head node section). The alert is something that comes up automatically when the user selects multiple instance types

Reply:
Gotcha!